ndk-build/src/cargo.rs

```diff
@@ -3,6 +3,10 @@ use crate::ndk::Ndk;
 use crate::target::Target;
 use std::process::Command;
 
+pub trait Env {
+    fn var(&self, key: &str) -> std::result::Result<std::borrow::Cow<'_, str>, std::env::VarError>;
```

Comment:
We should add add a `.cargo/config.toml`, disallow `-Wunused-qualifications` and do a pass, worked out for some other repos :)

(It won't find long paths, but it will tell you when they also already happen to be imported in scope. Is there any standard detailing when to import in scope and when to use [partial] paths?)